src/components/view/AuthorizedRoute/index.js

```diff
@@ -25,7 +25,7 @@ class AuthorizedRoute extends Component {
         }
 
         if (userPermissions && userPermissions.basic && location.pathname === '/') {
-            return <Redirect to="/intake"/>
+            return <Redirect to="/contact"/>
```

Comment:
I'm thinking you can remove this whole if statement, because app.js has it setup such that if the user navigates to "/" they're going to get routed to the Contact component anyway. on line 119 of App.js

```javascript
<Switch>
  <AuthorizedRoute exact path="/contact/" component={Contact} />
  <AuthorizedRoute path="/contact/:uid/:action" component={Contact} />
  <AuthorizedRoute exact path="/reports" component={Reports} />
  <AuthorizedRoute path="/" component={Contact} />
</Switch>
```

Reply:
Good call